src/applications/static-pages/static-pages-entry.js

```diff
@@ -79,59 +37,79 @@ Raven.context(
 
 createAdditionalInfoWidget();
 
-if (pensionPages.has(location.pathname)) {
+if (
+  document.querySelector(
+    `[data-widget-type="${widgetTypes.PENSION_APP_STATUS}"]`,
+  )
+) {
```

Comment:
Are these ifs necessary? It seems all of the `create*` functions already do something like:

```
const root = document.querySelector(...);
if (root) { ... }
```

Reply:
It probably isn't that expensive, but maybe we can save ourselves one DOM search for each widget.

Reply:
Yeah, I can't see a reason why we shouldn't do that, now that we have a unique type to check.